CIPs/CIP-85/CIP-85.md

```diff
@@ -86,80 +82,86 @@ The `CollectionSlice` schema must be an `object` with a `$ceramic` field from [C
   title: 'MyCollectionSLice',
   type: 'object',
   properties: {
+    collection: { type: 'string', maxLength: 150 },
+    sliceNumber: { type: 'integer', minimum: 0 },
     contents: {
       type: 'array',
       maxItems: 256,
-      minItems: 1,
+      minItems: 0,
       items: {
         oneOf: [{ type: 'object', ... }, { type: 'null' }],
       },
     },
-    next: { type: 'string', maxLength: 150 },
-    prev: { type: 'string', maxLength: 150 },
   },
-  required: ['contents'],
+  required: ['collection', 'sliceNumber', 'contents'],
 }
 ```
 
 ### Algorithms
 
+#### Deterministic slice access
+
+Accessing slices in the collection relies on Ceramic's ability to load documents deterministically based on their genesis contents:
+
+- `collection`: the collection's `DocID` string
+- `sliceNumber`: the slice's number, an integer between `0` (first slice) and `slicesCount - 1` (last slice)
+- `contents`: empty array
+
 #### First insertion
 
 > This flow assumes a prerequisite check that the `AppendCollection` document has not been created yet.
 
 1. Create the `CollectionSlice` document with the `contents` array containing the item to insert.
```

Comment:
This needs to first be created with empty `contents` array then updated to add the item

Reply:
Good catch, thanks! I updated it.